rbac/management/role/definer.py

```diff
@@ -83,14 +83,23 @@ def seed_roles(tenant):
         for f in os.listdir(roles_directory)
         if os.path.isfile(os.path.join(roles_directory, f)) and f.endswith(".json")
     ]
+    current_role_ids = set()
     with tenant_context(tenant):
         with transaction.atomic():
             for role_file_name in role_files:
                 role_file_path = os.path.join(roles_directory, role_file_name)
                 with open(role_file_path) as json_file:
                     data = json.load(json_file)
                     role_list = data.get("roles")
-                    _update_or_create_roles(tenant, role_list)
+                    role = _update_or_create_roles(tenant, role_list)
+                    current_role_ids.add(role.id)
+
+        roles_to_delete = Role.objects.filter(system=True).exclude(id__in=current_role_ids)
+        logger.info(
+            f"[READ-ONLY] Deleting the following '{roles_to_delete.count()}' roles(s): {roles_to_delete.values()}"
```

Comment:
I think this is the right idea, but the log message may be unclear to someone seeing it go by. Maybe worth changing `Deleting the following roles` to something like `The following {count} roles eligible for removal` or altering the `[READ-ONLY]` to WIP or [TESTING]?

Reply:
Good call. It's confusing, I'll update! Thanks!